apidef/api_definitions.go

```diff
@@ -326,8 +326,12 @@ type APIDefinition struct {
 		AllowedAuthorizeTypes  []osin.AuthorizeRequestType `bson:"allowed_authorize_types" json:"allowed_authorize_types"`
 		AuthorizeLoginRedirect string                      `bson:"auth_login_redirect" json:"auth_login_redirect"`
 	} `bson:"oauth_meta" json:"oauth_meta"`
-	Auth                          Auth                 `bson:"auth" json:"auth"`
-	UseBasicAuth                  bool                 `bson:"use_basic_auth" json:"use_basic_auth"`
+	Auth         Auth `bson:"auth" json:"auth"`
+	UseBasicAuth bool `bson:"use_basic_auth" json:"use_basic_auth"`
+	BasicAuth    struct {
+		DisableCaching bool `bson:"enable_caching" json:"enable_caching"`
```

Comment:
Should be `EnableCaching` ?

Reply:
We want caching to be enabled by default - so a user needs to disable caching explicitly

Reply:
Actually looking at rest of code i think the bson and json tags need changing to `disable_caching`